theforeman.org/pipelines/release/source/smart-proxy.groovy

```diff
@@ -14,9 +14,9 @@ pipeline {
                 git url: git_url, branch: git_ref
             }
         }
-        stage("test ruby-2.7") {
+        stage("test ruby") {
             steps {
-                run_test(ruby: '2.7')
+                run_test(ruby: '2.7', '3.0', '3.1')
```

Comment:
We missed in the review, but this is not valid. https://ci.theforeman.org/job/smart-proxy-develop-source-release/175/console now has:
> java.lang.NoSuchMethodError: No such DSL method 'run_test' found among steps

This is because there's no method that fits the method signature.

What should have been done was to convert the stage to a matrix, like in the other file.

Reply:
@ekohl https://github.com/theforeman/jenkins-jobs/pull/324